### PR TITLE
Add section that displays a random quote with an animation

### PR DESCRIPTION
**Pull request by AnudeepYakkala**
I added a section that randomly chooses a quote from a list of my favorite quotes and displays it with a text animation when the "generate" button is clicked. Previously I had it choosing from random Star Wars quotes but I thought that changing it to choose from some of my favorite inspirational quotes would make it more personal.  

The following is a screenshot of the new feature:
image

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -9,13 +9,17 @@
...
   <body>
-    <div id="content">
-      <h1>Anudeep's Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random Star Wars quote:</p>
-      <button onclick="addRandomStarWarsQuote()">Generate</button>
-      <div id="quote-container"></div>
-      <br>
+    <h1>Anudeep's Portfolio</h1>
+
```
Comment (chen-dawn) [inline portfolio/src/main/webapp/index.html:13]: Extra line here

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -206,9 +210,28 @@ <h5>Mount Diablo</h5>
       </div>
     </div>
 
-    <!-- Bootstrap JS -->
+    <!-- Footer -->
+    <div id="footer-container">/
+        <a href="https://linkedin.com/in/ayakkala/">
+          <img class="logo-image" src="images/linkedIn-logo.png">
```
Comment (chen-dawn) [inline portfolio/src/main/webapp/index.html:216]: Add alt text to all images

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -9,13 +9,17 @@
...
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random Star Wars quote:</p>
-      <button onclick="addRandomStarWarsQuote()">Generate</button>
-      <div id="quote-container"></div>
-      <br>
+    <h1>Anudeep's Portfolio</h1>
+
+
+    <!-- Random Quote -->
+    <div id="quote-background">
```
Comment (chen-dawn) [inline portfolio/src/main/webapp/index.html:16]: Good job with the IDs!

File: portfolio/src/main/webapp/script.js [hunk 4]
```diff
@@ -13,17 +13,35 @@
 // limitations under the License.
 
 /**
- * Adds a random Star Wars Quote to the page.
+ * Adds a random quote to the page.
  */
-function addRandomStarWarsQuote() {
+function addRandomQuote() {
   const quotes =
-      ['Help me, Obi-Wan Kenobi. You are my only hope. -Lelia Organa', 'The force will be with you. Always. -Obi Wan Kenobi',
```
Comment (chen-dawn) [inline portfolio/src/main/webapp/script.js:20]: Awww I liked the star wars quotes :(

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (AnudeepYakkala) [reply portfolio/src/main/webapp/index.html:13]: Oops. I'll remove it.

File: portfolio/src/main/webapp/index.html (hunk 2 above)
Comment (AnudeepYakkala) [reply portfolio/src/main/webapp/index.html]: I think Li mentioned this in a previous pull request. It should be fixed once I merge with master.

File: portfolio/src/main/webapp/index.html (hunk 3 above)
Comment (AnudeepYakkala) [reply portfolio/src/main/webapp/index.html:16]: Thanks!

File: portfolio/src/main/webapp/script.js (hunk 4 above)
Comment (AnudeepYakkala) [reply portfolio/src/main/webapp/script.js:20]: I did too. I think I will add them back along with the other quotes I added.

**Pull request merged**